pkg/controller/servicebindingrequest/annotations/annotations.go

```diff
@@ -0,0 +1,72 @@
+package annotations
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/dynamic"
+)
+
+// bindingType encodes the medium the binding should deliver the configuration value.
+type bindingType string
+
+const (
+	// BindingTypeVolumeMount indicates the binding should happen through a volume mount.
+	BindingTypeVolumeMount bindingType = "volumemount"
+	// BindingTypeEnvVar indicates the binding should happen through environment variables.
+	BindingTypeEnvVar bindingType = "env"
+)
+
+// supportedBindingTypes contains all currently supported binding types.
+var supportedBindingTypes = map[bindingType]bool{
+	BindingTypeVolumeMount: true,
+	BindingTypeEnvVar:      true,
+}
+
+// dataPath is the path ConfigMap and Secret resources use to scope their data.
+var dataPath = "data"
```

Comment:
The API that consumes `dataPath` uses it to get a pointer to the value, so it'd require the rest of the API to be modified accordingly, which means creating a copy of the constant to be able to use its reference.

Reply:
@isutton the conversation has been resolved, but please state what is the outcome? it was not addressed here, so do we have a ticker where this will be touched?

Reply:
I considered this change irrelevant for the well functioning of the refactoring. Please create an issue if you'd like to take this further.